start.sh

```diff
@@ -1,9 +1,35 @@
 #!/bin/bash
-IMAGE_NAME="torchserve:1.0"
+IMAGE_NAME="pytorch/torchserve:latest"
 
-echo "Starting torchserve:1.0 docker image"
+for arg in "$@"
+do
+    case $arg in
+        -h|--help)
+          echo "options:"
+          echo "-h, --help  show brief help"
+          echo "-g, --gpu specify to use gpu"
+          echo "-d, --gpu_devices to use specific gpu device ids"
+          exit 0
+          ;;
+        -g|--gpu)
+          DOCKER_RUNTIME="--runtime=nvidia"
```

Comment:
@dhanainme The new way of running docker with gpus is with --gpus flag. Please change to that as described in https://github.com/pytorch/serve/issues/191

Reply:
Was trying to use this originally, But the new format uses a very weird quoting for multiple GPUs which does not play well with bash. Would prefer leaving it this way.

docker run --rm -it --gpus '"device=0,1,2"' -p 8080:8080 -p 8081:8081 torchserve:v0.1-gpu-latest